console/console.proto

```diff
@@ -425,6 +433,11 @@ message StorageList {
   repeated nakama.api.StorageObject objects = 1;
   // Approximate total number of storage objects.
   int32 total_count = 2;
+  // Avaliable collection names in the whole of the storage
+  repeated string collections = 3;
```

Comment:
This needs to be part of another API call to return only the collection data - so we can power the dropdown.

Reply:
Also typo in "Avaliable"